packages/wells/src/__tests__/testUtils.ts

```diff
@@ -3,14 +3,18 @@
 import { name } from '../../package.json';
 import { WELL_SERVICE_BASE_URL } from '../client/api/utils';
 import CogniteWellsClient from '../client/CogniteWellsClient';
+import { Cluster } from '../client/model/Cluster';
 
 export function setupClient(
   baseUrl: string = WELL_SERVICE_BASE_URL
 ): CogniteWellsClient {
-  return new CogniteWellsClient({
-    appId: `JS WELLS SDK integration tests (${name})`,
-    baseUrl,
-  });
+  return new CogniteWellsClient(
+    {
+      appId: `JS WELLS SDK integration tests (${name})`,
+      baseUrl,
+    },
+    Cluster.PROD
+  );
```

Comment:
ps, I removed baseUrl from the CogniteWellsClient constructor. We're already setting that inside of the class itself